src/engine/ruleset/decoders/fortinet/fortimanager.yml

```diff
@@ -0,0 +1,201 @@
+      - destination.nat.port : $~kvMap.tranport
+      - destination.port : $~kvMap.dstport
+      - event.code: $~kvMap.logid
+      - log.level: $~kvMap.level
+      - network.bytes : $~kvMap.sentbyte
+      - network.bytes : +i_calc/sum/$~kvMap.rcvdbyte
+      - network.direction : $~kvMap.direction
+      - observer.egress.interface.name : $~kvMap.dstintf
+      - observer.ingress.interface.name : $~kvMap.srcintf
+      - observer.serial_number: $~kvMap.devid
+      - related.hosts: +a_append/$~kvMap.devname
+      - related.ip: +a_append/$~kvMap.dstip
+      - related.ip: +a_append/$~kvMap.srcip
+      - related.ip: +a_append/$~kvMap.tranip
+      - rsa.internal.messageid: generic_fortinetmgr_1
+      - rsa.misc.action: +a_append/$~kvMap.action
+      - rsa.misc.context: $~kvMap.trandisp
+      - rsa.misc.fcatnum: $~kvMap.cat
+      - rsa.misc.filter: $~kvMap.catdesc
+      - rsa.misc.hardware_id: $~kvMap.device_id
+      - rsa.misc.log_session_id: $~kvMap.session_id
+      - rsa.misc.obj_name: $~kvMap.app
+      - rsa.misc.policy_id: $~kvMap.policyid
+      - rsa.misc.reference_id: $~kvMap.logid
+      - rsa.misc.vsys : $~kvMap.vd
+      - rsa.network.interface : $~kvMap.dstintf
+      - rsa.network.network_service : $~kvMap.service
+      - rsa.network.sinterface : $~kvMap.srcintf
+      - rsa.time.duration_time : $~kvMap.duration
+      - rsa.time.event_time_str: $~kvMap.eventtime
+      - rsa.web.reputation_num: $~kvMap.crscore
+      - source.bytes: $~kvMap.sentbyte
+      - source.geo.country_name: $~kvMap.srccountry
+      - source.ip : $~kvMap.srcip
+      - source.port: $~kvMap.srcport
+
+  # event_fortinetmgr  (hasn't signature_id and hasn't vd)
+  - check:
+    - ~kvMap.signature_id: +ef_not_exists
+    - ~kvMap.vd: +ef_not_exists
+    map:
+      - event.code: $~kvMap.log_id
+      - log.level: $~kvMap.pri
+      - observer.serial_number: $~kvMap.device_id
+      - related.hosts: +a_append/$~kvMap.adom
+      - related.hosts: +a_append/$~kvMap.devname
+      - related.user: +a_append/$~kvMap.user
+      - rsa.db_index: $~kvMap.msg
+      - rsa.internal.event_desc: $~kvMap.desc
+      - rsa.internal.messageid: event_fortinetmgr
+      - rsa.misc.action: +a_append/$~kvMap.action
+      - rsa.misc.hardware_id: $~kvMap.device_id
+      - rsa.misc.log_session_id: $~kvMap.session_id
+      - rsa.misc.reference_id: $~kvMap.log_id
+      - server.domain : $~kvMap.adom
+      - user.name : $~kvMap.user
+      - rsa.network.domain: $server.domain
+
+  # Checks if a valid fortinet log was found
+  - check: rsa.internal.messageid==generic_fortinetmgr OR rsa.internal.messageid==generic_fortinetmgr_1 OR rsa.internal.messageid==event_fortinetmgr
+    map:
+      - "@timestamp": +s_concat/$~kvMap.date/T/$~kvMap.time/.000Z
+      - event.action: $~kvMap.action
+      - event.dataset: "fortinet.fortimanager"
+      - event.module: "fortinet"
+      - fileset.name: fortimanager
+      - input.type: log
+      - network.protocol: $~kvMap.proto
```

Comment:
Don't map to an ECS field without normalization. Below test the conditions on the temporal field. This is a best practice, it's preferred to do not to have an ECS field instead of one with wrong mappings.